README.md

```diff
@@ -72,19 +67,10 @@ vars:
 
 For additional configurations for the source models, such as the company and tag variables, visit the [Intercom source package](https://github.com/fivetran/dbt_intercom_source).
 
-## Contributions and Future Considerations
-Currently Intercom V2.0 does not support API exposure to company defined business hours. As such, 
-all time_to metrics are calculated in their entirety without subtracting out business hours. A feature 
-ticket has been logged with Intercom to provide API exposure for business hours in a later release.
-Don't see a model or specific metric you would have liked to be included? Notice any bugs when installing 
-and running the package? If so, we highly encourage and welcome contributions to this package! 
-Please create issues or open PRs against `master`. Check out [this post](https://discourse.getdbt.com/t/contributing-to-a-dbt-package/657) 
+## Contributions
+Additional contributions to this package are very welcome! Please create issues or open PRs against `master`. Check out [this post](https://discourse.getdbt.com/t/contributing-to-a-dbt-package/657) 
 on the best workflow for contributing to a package.
 
-## Database support
```

Comment:
We typically don't include supported destination info in the READMEs because our [dbt documentation](https://fivetran.com/docs/transformations/dbt#fivetrandbtpackages) already states that our packages are compatible with BigQuery, Snowflake and Redshift. 

As for Spark - see my note above about future considerations.

Reply:
Makes sense, thank you!

Reply:
@fivetran-chloe In the past, we've included destination support in all of our READMEs -- Although we're including this in our docs, I think it's pretty important to keep in the READMEs as we receive questions about this regularly and people don't necessary think to go to our docs if they come through via dbt hub. 

Regarding Spark - I agree, let's remove that.

Reply:
@fivetran-kristin I've added the database support section back in, but removed the Spark reference.